locales/en-US/snoozetabs.properties

```diff
@@ -61,3 +61,30 @@ manageUndo=Undo
 # Displayed in the Manage Snoozed Tabs menu when tab is set to snooze next time
 # Firefox opens
 manageDateNextOpen=Next time<br>Firefox opens
+
+calendar_today=Today
+calendar_now=Now
+calendar_backToToday=Back to today
+calendar_ok=Ok
+calendar_clear=Clear
+calendar_month=Month
+calendar_year=Year
+calendar_timeSelect=Select time
+calendar_dateSelect=Select date
+calendar_monthSelect=Choose a month
+calendar_yearSelect=Choose a year
+calendar_decadeSelect=Choose a decade
+calendar_yearFormat=YYYY
+calendar_dateFormat=M/D/YYYY
+calendar_dayFormat=D
+calendar_dateTimeFormat=M/D/YYYY HH:mm:ss
+calendar_monthFormat=MMMM
+calendar_monthBeforeYear=true
+calendar_previousMonth=Previous month (PageUp)
+calendar_nextMonth=Next month (PageDown)
+calendar_previousYear=Last year (Control + left)
+calendar_nextYear=Next year (Control + right)
```

Comment:
I feel like these are going to need a localizer's note to let them know not to translate the stuff in the `()`s.